Cargo.toml

```diff
@@ -15,7 +15,22 @@ crate-type = ["cdylib"]
 [workspace]
 members = [
     "oxidation/libparsec",
-    "oxidation/libparsec/crates/*",
+    "oxidation/libparsec/crates/client",
+    "oxidation/libparsec/crates/client_connection",
+    "oxidation/libparsec/crates/client_high_level_api",
+    "oxidation/libparsec/crates/core",
+    "oxidation/libparsec/crates/core_fs",
+    "oxidation/libparsec/crates/crypto",
+    "oxidation/libparsec/crates/platform_async",
+    "oxidation/libparsec/crates/platform_device_loader",
+    "oxidation/libparsec/crates/platform_storage",
+    "oxidation/libparsec/crates/platform_storage2",
+    "oxidation/libparsec/crates/protocol",
+    "oxidation/libparsec/crates/serialization_format",
+    "oxidation/libparsec/crates/testbed",
+    "oxidation/libparsec/crates/tests_fixtures",
+    "oxidation/libparsec/crates/tests_macros",
+    "oxidation/libparsec/crates/types",
```

Comment:
The `oxidation/libparsec/crates/*` is simpler to work with, Why do you choose to explicitly list every crates ?